Automattic/jetpack
# Charts: Adding multi series support for bar chart

Fixes https://github.com/Automattic/jetpack-roadmap/issues/2054

## Proposed changes:

* extending bar charts to support multiple data series
* extending tooltips to accept custom inner content
* updating example data for theme story 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* navigate to the directory and run `storybook: cd projects/js-packages/charts` and `pnpm run storybook`
* verify that the bar chart now shows multiple series in the default story
* verify that other charts are working

![SCR-20241217-oawg](https://github.com/user-attachments/assets/c2b20a21-b173-41c8-9b44-fe0d0b24ba17)
